.circleci/config.yml

```diff
@@ -31,12 +31,25 @@ executors:
     resource_class: xlarge
 
 commands:
+  save_merge_commit:
+    steps:
+      - save_cache:
+          name: 'Save Merge Commit'
+          key: git-{{ .Branch }}-{{ .Revision }}
+          paths:
+            - .merge_sha
```

Comment:
Renamed to `.CIRCLECI_WORKFLOW_MERGE_COMMIT`. We cannot use `.circleci/*` because the directory doesn't exist during fetch because we didn't clone the repo.